### PR TITLE
serving Llama 70b on 2 x  L4 GPU

### PR DESCRIPTION
**Pull request by samos123**
Tutorial on how to use text-generation-inference with Llama 2 70B Chat

### CODE REVIEW

File: serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml
Comment (danielvegamyhre) [inline serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml:28]: It might be worth adding a short sentence explaining what quantization is and why we are doing it for inference, with a link to more detailed information if the user is interested in learning more.

File: serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml
Comment (danielvegamyhre) [inline serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml:24]: It might be helpful to give a brief bullet point on what some of these less self-explanatory environment variables do, such as `NUM_SHARD`, so people can adapt this to other inference infra setups. For example, if you try googling `text-generation-interface \"NUM_SHARD\"` I don't see any relevant docs about this env var (maybe I'm just not finding them), and so the user will be forced to read the [comments](https://github.com/huggingface/text-generation-inference/blob/v1.0.3/launcher/src/main.rs#L119-L124) in the main.rs file for answers, which is not ideal.

**Comment by danielvegamyhre** [comment]
@samos123 Added a few minor comments but otherwise LGTM!

File: serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml
Comment (samos123) [reply serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml]: I didn't find any good blogs that I would link to so I rather leave that up to the user to figure out. I did add some comments on why we use quantization and explained it.

File: serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml
Comment (samos123) [reply serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml]: I updated with some basic explanation.

**Comment by samos123** [comment]
@danielvegamyhre could you please review again?

**Comment by danielvegamyhre** [comment]
/lgtm

cc @richardsliu who has approval powers in this repo

File: serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml
Comment (ahg-g) [reply serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml]: I think we should at least explain why we selected this value.

Comment (samos123) [reply serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml]: I added an explanation already and also added explanation on how this setting impacts how much GPU memory is needed

**Review by richardsliu — APPROVED** [review]

File: tutorials/serving-llama2-70b-on-l4-gpus/README.md
Comment (samos123) [reply tutorials/serving-llama2-70b-on-l4-gpus/README.md:114]: weird, I'm not sure why I never found that doc before. I've added the link to the PR.

**Pull request merged**
